server/src/main/java/org/opensearch/gateway/remote/ClusterStateDiffManifest.java

```diff
@@ -344,12 +313,6 @@ public static ClusterStateDiffManifest fromXContent(XContentParser parser, long
                         currentFieldName = parser.currentName();
                         parser.nextToken();
                         switch (currentFieldName) {
-                            case UPSERTS_FIELD:
-                                builder.indicesRoutingUpdated(convertListToString(parser.listOrderedMap()));
-                                break;
-                            case DELETES_FIELD:
-                                builder.indicesRoutingDeleted(convertListToString(parser.listOrderedMap()));
-                                break;
                             case DIFF_FIELD:
```

Comment:
Do we need to bump up the CODEC version for BWC?

Reply:
Made changes to increment CODEC version.